### PR TITLE
Create DV17.md

### PR DESCRIPTION
**Pull request by rveltz**

### CODE REVIEW

File: _publications/DV17.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+---
+In this work, we provide three different numerical evidences for the occurrence of a Hopf bifurcation in a recently derived [De Masi et al., J. Stat. Phys. 158, 866–902 (2015) and Fournier and locherbach, Ann. Inst. H. Poincar􏰲e Probab. Stat. 52, 1844–1876 (2016)] mean field limit of a stochastic network of excitatory spiking neurons. The mean field limit is a challenging nonlocal nonlinear transport equation with boundary conditions. The first evidence relies on the computation of the spectrum of the linearized equation. The second stems from the simulation of the full mean field. Finally, the last evidence comes from the simulation of the network for a large number of neurons. We provide a “recipe” to find such bifurcation which nicely complements the works in De Masi et al. [J. Stat. Phys. 158, 866–902 (2015)] and Fournier and l€ocherbach [Ann. Inst. H. Poincar􏰲e Probab. Stat. 52, 1844–1876 (2016)]. This suggests in return to revisit theoretically these mean field equations from a dynamical point of view. Finally, this work shows how the noise level impacts the transition from asynchronous activity to partial synchronization in excitatory globally pulse-coupled networks
```
Comment (ChrisRackauckas) [inline _publications/DV17.md]: `Poincar􏰲e` seems to have a unicode error? Also `l€ocherbach`?

**Comment by rveltz** [comment]
Thank you. I corrected it.

File: _publications/DV17.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+---
+In this work, we provide three different numerical evidences for the occurrence of a Hopf bifurcation in a recently derived [De Masi et al., J. Stat. Phys. 158, 866–902 (2015) and Fournier and löcherbach, Ann. Inst. H. Poincaré Probab. Stat. 52, 1844–1876 (2016)] mean field limit of a stochastic network of excitatory spiking neurons. The mean field limit is a challenging nonlocal nonlinear transport equation with boundary conditions. The first evidence relies on the computation of the spectrum of the linearized equation. The second stems from the simulation of the full mean field. Finally, the last evidence comes from the simulation of the network for a large number of neurons. We provide a “recipe” to find such bifurcation which nicely complements the works in De Masi et al. [J. Stat. Phys. 158, 866–902 (2015)] and Fournier and löcherbach [Ann. Inst. H. Poincaré Probab. Stat. 52, 1844–1876 (2016)]. This suggests in return to revisit theoretically these mean field equations from a dynamical point of view. Finally, this work shows how the noise level impacts the transition from asynchronous activity to partial synchronization in excitatory globally pulse-coupled networks
```
Comment (ChrisRackauckas) [inline _publications/DV17.md]: Capital on `löcherbach`? Two places.

**Comment by rveltz** [comment]
Thank you!

**Pull request merged**